StitchCoreServicesTwilio-iOS/StitchCoreServicesTwilioTests-iOS/Stitch_iOSServiceTwilioTests.swift

```diff
@@ -0,0 +1,113 @@
+import XCTest
+import StitchCoreAdminClient
+import StitchCore_iOS
+import StitchCore
+import StitchCoreTestUtils_iOS
+import StitchCoreServicesTwilio_iOS
+import StitchCoreServicesTwilio
+
+class Stitch_iOSServiceTwilioTests: BaseStitchIntTestCocoaTouch {
+    private let twilioSidProp = "test.stitch.twilioSid"
+    private let twilioAuthTokenProp = "test.stitch.twilioAuthToken"
+    
+    private func fetchPList() -> [String: Any]? {
```

Comment:
[q] How would I set this up myself if I wanted to test this PR? I don't know the steps to run this test.